### ISSUE TITLE
[A11Y]Remove title attributes from Send and attachment buttons

### ISSUE THREAD

**Issue by elopezanaya** [microsoft/BotFramework-WebChat#4712]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

From FORD revision and assessment ->

"Title attributes are not widely supported among assistive technologies and should not be used in order to provide control labels. Additionally, having a title attribute as well as an aria-label can cause duplicate announcements when using assistive technologies"

Recommendation -> 

Remove title attributes in buttons send and attachments, and replace with aria-labels (if aria-label is not present)

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Send box

### How to reproduce the issue?

1.launch the chat 
2.- enable dev-tools to check on the resource for the UI

### What do you expect?

just aria-label in the buttons 

### What actually happened?

title values are present

**Comment by elopezanaya** [comment]
any update on this ask?

**Comment by compulim** [comment]
I need to check with accessibility team.

I think "title" should be used in preference over "aria-label". "aria-label" should only be used if tooltip is undesirable. But in this case, the tooltip is not undesirable. 

If you look at Fluent UI button, they also put a tooltip on the side the button, https://react.fluentui.dev/?path=/docs/components-button-button--default. That means, "button with tooltip" is not commonly an undesirable thing.

Also, the bug say both "aria-label" and "title" are both set, which is wrong.

Excerpt from WAI-ARIA 1.3, https://w3c.github.io/aria/#aria-label.

> Most host languages provide an attribute that could be used to name the element (e.g., the [title](https://html.spec.whatwg.org/multipage/dom.html#attr-title) attribute in HTML), yet this could present a browser tooltip. In the cases where DOM content or a tooltip is undesirable, authors MAY set the accessible name of the element using [aria-label](https://w3c.github.io/aria/#aria-label), if the element does not [prohibit](https://w3c.github.io/aria/#prohibitedattributes) use of the attribute.

**Comment by compulim** [comment]
Given Fluent UI and Teams are using `title` for icon buttons and we did not violate any particular rules in WCAG.

I am closing this bug as won't fix.
